### PR TITLE
[civ2][lint/5] migrate all but annotations over to civ02

### PR DESCRIPTION
**Pull request by can-anyscale**
Migrate the rest of lint over to civ02, except for annotations since that depends on ray installation.

Test:
- CI

### CODE REVIEW

File: .buildkite/lint.rayci.yml [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
 
-  - label: "lint: doc readme"
+  - label: ":book: lint: doc readme"
     commands:
       - pip install -c python/requirements_compiled.txt docutils
       - cd python && python setup.py check --restructuredtext --strict --metadata
     depends_on: forge
 
-  - label: "lint: dashboard format"
+  - label: ":book: lint: dashboard format"
```
Comment (aslonnie) [inline .buildkite/lint.rayci.yml:32]: this is probably not `:book:` ? I think `:book:` means it is related to docs?

File: ci/lint/check-bazel-buildifier.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Checks bazel buildifier format
+
+set -euxo pipefail
+
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE:-$0}")" || exit; pwd)/.."
```
Comment (aslonnie) [inline ci/lint/check-bazel-buildifier.sh:6]: I am not sure if there are really people that will attempt to run these lint script locally.

**Review by aslonnie — APPROVED** [review]

File: ci/lint/check-bazel-buildifier.sh (hunk 2 above)
Comment (can-anyscale) [reply ci/lint/check-bazel-buildifier.sh:6]: probably no-one

**Comment by can-anyscale** [comment]
w00t there are no ci jobs running on this PR?

**Pull request merged**